lib/widgets/enable_backup_dialog.dart

```diff
@@ -258,13 +258,9 @@ class _BackupNowButtonState extends State<_BackupNowButton> {
     BackupProvider selectedProvider,
   ) async {
     try {
-      if (selectedProvider.isRemoteServer) {
-        await _enterRemoteServerCredentials();
-      } else {
-        await _backupNow(
-          widget.backupSettings.copyWith(backupProvider: selectedProvider),
-        ).then((_) => Navigator.pop(context));
-      }
+      await _backupNow(
```

Comment:
Don't we want users to be able to switch their remote server, re-enter their auth credentials here?

Reply:
I now see why this is required, we do prompt for auth data on Select Backup Provider dialog.